### DISCUSSION TITLE
Add AppBar Horizontal Orientation

### DISCUSSION THREAD

**Discussion by erickmiyazaki** [microsoft/fluentui-blazor#2752] [Ideas]
Hi, 

Could we add Horizontal Orientation to the AppBar?

I think this would be great when implementing Web Apps for mobile devices.
Just like in Teams Mobile App, where the AppBar is horizontal and sits at the bottom of the screen.

**Comment by vnbaaij** [comment]
Working on it...
![AppBar-horizontal](https://github.com/user-attachments/assets/48adf37f-62b8-4ef6-bd27-a4bf7203bca8)

Still needs some polishing before it will be ready for being included though. No promises on when it will be available.

**Reply by SpenceDahl** [reply]
Thank you!

**Reply by erickmiyazaki** [reply]
Thank you!
That would be great!
